### ISSUE TITLE
How are schools dealt with under FOI

### ISSUE THREAD

**Issue by benrfairless** [openaustralia/righttoknow#463]
I'm not sure if each school would be covered individually under FOI, as teachers in public schools are normally employed not by the school but by the Department/Directorate of Education.

I can't see anything in the law which makes it clear if schools == authorities. In the ACT, an FOIable authority either needs to be specified via the FOI regulations or be an "administrative unit" which is established by Administrative Arrangements.

The Department of Education is FOIable, and the Minister has responsibility for government and non-government schools, but I don't think each individual school is covered under FOI.

@RichardTaylor I'm not sure if you have any additional insight which you could offer here?

**Comment by benrfairless** [comment]
I've spoken with Margaret from the ACT Government to clarify how the directorate would process a request. Generally speaking, schools would refer the request to the Directorate anyway and private schools aren't covered under FOI.

Suggest we:
- [ ] Mark all schools as `not_apply`
- [ ] Add a note to every school that, if they want to, they can request information from the ACT-DET

Are there any objections @openaustralia/team-right-to-know @openaustralia/owners

**Comment by henare** [comment]
@brfairless any reason to not just delete the schools?

**Comment by benrfairless** [comment]
I was thinking if people search for their school, it would be better to at least list it with instructions on how to make a request to DET, but I'm just as happy to delete them.

**Comment by henare** [comment]
Yeah it could also help Google juice too I guess. Let's have a think about it.

**Comment by henare** [comment]
I've split out the question of what to do with the ACT into #482.

This issue is still open since we need to work out what the story is with schools in other states.

**Comment by henare** [comment]
Some hospitals in Victoria come under the banner of a district health authority. I included their names in the notes field so they're still searchable but all the requests will go to the one authority.

This probably wouldn't work for schools since there'd be heaps listed under the department of education.
